### DISCUSSION TITLE
Present Roadmap - Update

### DISCUSSION THREAD

**Discussion by chempro** [dotnet/winforms#5428] [General]
At the moment it’s really hard to find out the current status of where Winforms are with .NET
The current roadmap is really wishy washy and doesn’t provide any real information https://github.com/dotnet/winforms/blob/main/docs/roadmap.md#present

I’ve searched on .NET blogs and the most recent post only mentions changes to the default font… https://devblogs.microsoft.com/dotnet/whats-new-in-windows-forms-in-net-6-0-preview-5/

Going further back I did find https://devblogs.microsoft.com/dotnet/windows-forms-designer-for-net-core-released/ but that is from more than a year ago

It would be good to have some rough timeframes on when the designer will be released (no longer preview) and when third party controls will be supported https://supportcenter.devexpress.com/ticket/details/t979482/design-time-support-for-devexpress-winforms-controls-for-net-5-faq

I’m not asking for rough timeframes to try and pressure, it’s so that we can plan our work i.e. roughly 3 months away is very different to roughly 3 years away

I do understand that there are some workarounds like using preview and using the old designer, but like a lot of enterprises we wouldn’t look to start until there is a non-preview release and third party controls are supported

**Comment by RussKie** [comment]
Thank you for nudge. We will update the stale roadmap docs shortly.
